css/styles.css

```diff
@@ -1,62 +1,348 @@
+  text-align: center;
 }
 
 .task-form.show {
+  display: block;
+}
+
+.task-form input {
+  width: 50%;
+  padding: 10px;
+  margin-right: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  margin-bottom: 10px;
+}
+
+.task-form button {
+  background-color: #4a90e2;
+  color: #fff;
+  border: none;
+  padding: 10px 20px;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+}
+
+.task-form button:hover {
+  background-color: #357ab7;
+}
+
+.task-list {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 20px;
+}
+
+.task-card {
+  width: 382px;
+  height: 163px;
+  gap: 0px;
+  border-radius: 5px;
+  border: 1px solid #dde2ff;
+  opacity: 1;
+  background: #ffffff;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  position: relative;
+  transition: all 0.3s;
+}
+
+.task-card p {
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 28.13px;
+  text-align: left;
+  color: #32394b;
+  margin: 0 0 10px;
+  padding: 5px;
+  border-radius: 5px;
+}
+
+.task-card .created-at {
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16.41px;
+  text-align: left;
+  color: #bbbdd0;
+  margin-bottom: 10px;
+  padding: 5px;
+  border-radius: 5px;
+}
+
+.task-card .edit-actions {
+  display: flex;
+  justify-content: flex-start;
+  gap: 10px;
+  margin-top: 15px;
+}
+
+.task-card .task-actions {
   display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.task-card .task-buttons {
+  display: flex;
+  gap: 10px;
+}
+
+.task-card .task-actions button,
+.task-card .edit-actions button {
+  background: none;
+  border: none;
+  cursor: pointer;
+  font-size: 1.2em;
+  transition: color 0.3s;
+}
+
+.task-card .task-actions .btn-check {
+  color: #0bc375;
+}
+
+.task-card .task-actions .btn-check:hover {
+  color: #0bc375;
+}
+
+.task-card .task-actions .btn-edit {
+  color: #ffc107;
+}
+
+.task-card .task-actions .btn-edit:hover {
+  color: #e0a800;
+}
+
+.task-card .task-actions .btn-delete,
+.task-card .edit-actions .btn-delete {
+  color: #dc3545;
+}
+
+.task-card .task-actions .btn-delete:hover,
+.task-card .edit-actions .btn-delete:hover {
+  color: #c82333;
+}
+
+.task-card .edit-actions .btn-save {
+  color: #32394b;
+  border: none;
+  padding: 5px 10px;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+}
+
+.task-card .edit-actions .btn-save:hover {
+  background-color: #357ab7;
+}
+
+.task-card.isCompleted p {
+  text-decoration: line-through;
+  color: #28a745;
+}
+
+.task-card.isCompleted .created-at {
+  color: #28a745;
+}
+
+.edit-input {
+  width: 90%;
+  padding: 10px;
+  margin-bottom: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+}
+
+.error-message {
+  color: #dc3545;
+  margin: 10px 0;
+}
+
+.completed-tag {
+  padding: 5px 10px;
+  background-color: #7a8dfd;
+  color: #fff;
+  border-radius: 5px;
+  font-size: 0.9em;
+  display: inline-block;
+}
+
+#btnLoadMore,
+#btnShowLess {
```

Comment:
id has the highest priority for implementing styling in css. General practice is to use class for styling. If it is not required or created more priority over the any class style's, please declare class name and use it for styling.